File: 2490-circular-sentence/2490-circular-sentence.java
class Solution {
    public boolean isCircularSentence(String sentence) {
        int n = sentence.length();
        for (int i = 1; i < n - 1; i++) {
            if (sentence.charAt(i) == ' ' && sentence.charAt(i - 1) != sentence.charAt(i + 1)) {
                return false;
            }
        }
        return sentence.charAt(0) == sentence.charAt(n - 1);
    }
}

File: 2490-circular-sentence/2490-circular-sentence.cpp
class Solution {
public:
    bool isCircularSentence(string sentence) {
        int n = sentence.length();
        for(int i=0; i<n; i++){
            if(sentence[i]==' ' and sentence[i-1]!=sentence[i+1]){
                return false;
            }
        }
        return sentence[0]==sentence[n-1];
    }
};